routes/hot_feed.go

```diff
@@ -555,9 +557,30 @@ func (fes *APIServer) PopulateHotnessInfoMap(
 			// to posts from the specified look-back period without even looking up the post time stamp.
 			isCreatePost, postHashCreated := CheckTxnForCreatePost(txn)
 			if isCreatePost {
+				// We start with the creator's balance in computing the score
```

Comment:
This part isn't that important, it just makes the creator's balance factor into a post's score, which offers a slight improvement.